trace/translator/sfx.go

```diff
@@ -122,8 +122,8 @@ func SAPMSpanFromSFXSpan(sfxSpan *trace.Span) (span *jaegerpb.Span) {
 
 // SFXTagsToJaegerTags returns process tags and span tags from the SignalFx span tags, endpoint (remote), and kind
 func SFXTagsToJaegerTags(tags map[string]string, remoteEndpoint *trace.Endpoint, kind *string) ([]jaegerpb.KeyValue, []jaegerpb.KeyValue) {
-	processTags := make([]jaegerpb.KeyValue, 0, len(tags))
-	spanTags := make([]jaegerpb.KeyValue, 0, len(tags)+3)
+	processTags := make([]jaegerpb.KeyValue, 0, 3)
+	spanTags := make([]jaegerpb.KeyValue, 0, len(tags)+4)
```

Comment:
Could optimize this to allocate only one slice of len(tags) + 4 the maximum number of tags and use the initial part of the slice for spanTags and the last part for processTags.

I think may worth it, would like to hear what others think.